File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver){
        super(license, driver);
    }

    @Override
    public Integer getPassenger() {
        return this.passenger;
    }

    @Override
    public void setPassenger(Integer passengers) {
        if(passengers == 6){
            this.passenger = passengers;
        } else {
            System.err.println("La van debe permitir 6 pasajeros");
        }
    }
}

File: PHP/uberVan.php
<?php
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver){
        parent::__construct($license, $driver);
        // $this->typeCarAccepted = $typeCarAccepted;
        // $this->seatsMaterial = $seatsMaterial;
    }

    public function setPassenger($passenger){
        if ($passenger == 6){
            $this->passenger = $passenger;
        } else {
            throw new Error('El número de pasajeros debe ser de 6');
        }
    }
}
?>